packages/@misk/common/CHANGELOG.md

```diff
@@ -1,111 +1,137 @@
 # Change Log - @misk/common
 
-This log was last generated on Mon, 04 Nov 2019 14:27:10 GMT and should not be manually modified.
+This log was last generated on Tue, 14 Jan 2020 19:22:06 GMT and should not be manually modified.
+
+## 0.1.24
+
+Tue, 14 Jan 2020 19:22:06 GMT
+
+_Version update only_
```

Comment:
And this is not

Reply:
Re-ran linting but doesn't seem to have changed it. These files are changed by robots so I'm fine leaving it as is.